arm-documentdb/2015-04-08/examples/DocumentDBDatabaseAccountCreateMax.json

```diff
@@ -0,0 +1,88 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2015-04-08",
+    "subscriptionId": "subid",
+    "createUpdateParameters": {
+      "location": "westus",
+      "tags": {},
+      "properties": {
+        "kind": "GlobalDocumentDB",
```

Comment:
kind should outside of the properties object. It should one level up along with tags and location.